rebar.config

```diff
@@ -1,16 +1,29 @@
 %% -*- mode: erlang -*-
-{erl_opts, [debug_info, warnings_as_errors, {platform_define, "^[0-9]+",
-                                             namespaced_types}]}.
+{erl_opts, [
+    debug_info,
+    warnings_as_errors,
+    {platform_define, "^[0-9]+", namespaced_types}
+]}.
+
 {deps, [
-        {protobuffs, "0.8.*", {git, "git://github.com/basho/erlang_protobuffs.git", {tag, "0.8.1p5"}}}
-       ]}.
+    {protobuffs, "0.8.*", {git, "git://github.com/basho/erlang_protobuffs.git", {tag, "0.8.2"}}},
+    {hamcrest, ".*", {git, "https://github.com/hyperthunk/hamcrest-erlang.git", {branch, "master"}}}
```

Comment:
Without this dialyzer can't find one of the `hamcrest` types used by `meck` in testing.